cpu/cortexm_common/include/irq_arch.h

```diff
@@ -18,41 +18,59 @@
  * @}
  */
 
+
+#ifndef _IRQ_ARCH_H
+#define _IRQ_ARCH_H
+
 #include <stdint.h>
-#include "irq.h"
-#include "cpu.h"
+#include "cpu_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
 
 /**
  * @brief Disable all maskable interrupts
  */
-unsigned int irq_disable(void)
+static inline __attribute__((always_inline)) unsigned int irq_disable(void)
 {
     uint32_t mask = __get_PRIMASK();
+
```

Comment:
I guess this extra line can be removed.

Reply:
Yep will do once #14011 is in

Reply:
ping

Reply:
> ping

Sorry for the delay, Friday was a day off in France.

Regarding the extra line. I removed it just to remember why I had actually added it, when `uncrustifying ` that extra white line gets added, I think its better to leave it in because of that.